### ISSUE TITLE
SDK download fails for few languages

### ISSUE THREAD

**Issue by jsonds** [wso2/product-apim#2292]
**Description:**
The api store SDK download fails for the following languages. 

1. cpp
2. go
3. aspnet5
4. Spring
5. Haskel

cpp

```
TID: [-1234] [] [2018-01-08 17:39:06,577] ERROR {io.swagger.codegen.AbstractGenerator} -  cpprest/model-header.mustache (No such file or directory) {io.swagger.codegen.AbstractGenerator}
TID: [-1234] [] [2018-01-08 17:39:06,577] ERROR {io.swagger.codegen.AbstractGenerator} -  can't load template cpprest/model-header.mustache {io.swagger.codegen.AbstractGenerator}
TID: [-1234] [] [2018-01-08 17:39:06,578] ERROR {JAGGERY.site.blocks.sdk.ajax.sdk-create:jag} -  java.lang.RuntimeException: Could not generate model 'Error' {JAGGERY.site.blocks.sdk.ajax.sdk-create:jag}
```

go

```
TID: [-1234] [] [2018-01-08 17:45:31,321]  INFO {org.wso2.carbon.core.services.util.CarbonAuthenticationUtil} -  'admin@carbon.super [-1234]' logged in at [2018-01-08 17:45:31,321+0530] {org.wso2.carbon.core.services.util.CarbonAuthenticationUtil}
TID: [-1234] [] [2018-01-08 17:45:38,515] ERROR {io.swagger.codegen.AbstractGenerator} -  go/model.mustache (No such file or directory) {io.swagger.codegen.AbstractGenerator}
TID: [-1234] [] [2018-01-08 17:45:38,517] ERROR {io.swagger.codegen.AbstractGenerator} -  can't load template go/model.mustache {io.swagger.codegen.AbstractGenerator}
TID: [-1234] [] [2018-01-08 17:45:38,518] ERROR {JAGGERY.site.blocks.sdk.ajax.sdk-create:jag} -  java.lang.RuntimeException: Could not generate model 'ErrorListItem' {JAGGERY.site.blocks.sdk.ajax.sdk-create:jag}
```

**Suggested Labels:**
APIM 2.1.0 update 5 , Bug, Priority/Highest

**Affected Product Version:**
APIM 2.1.0 update 5

**OS, DB, other environment details and versions:**    
Ubuntu 14
Java version "1.8.0_51"
Google Chrome Version 63.0.3239.84 (Official Build) (64-bit)
DB - 5.5.58
